src/Nlog.RabbitMQ.Target/RabbitMQTarget.cs

```diff
@@ -174,6 +174,20 @@ public RabbitMQTarget()
 		/// </summary>
 		public ushort HeartBeatSeconds { get; set; } = 3;
 
+		/// <summary>
+		/// Add Global Diagnostic Context fields (https://github.com/NLog/NLog/wiki/Gdc-layout-renderer)
+		/// </summary>
+		public bool AddGdc { get; set; }
+		/// <summary>
+		/// Add Nested Diagnostics Logical Context fields (https://github.com/NLog/NLog/wiki/NDLC-Layout-Renderer)
+		/// </summary>
+		public bool AddMdlc { get; set; }
+
+		/// <summary>
+		/// Add Mapped Diagnostic Logical Context fields (https://github.com/NLog/NLog/wiki/MDLC-Layout-Renderer)
+		/// </summary>
+		public bool AddNdlc { get; set; }
```

Comment:
Consider renaming to IncludeNdlc like it is called in TargetWithContext

Reply:
I'll just remove it, because it's already included in TargetWithContext